bindings.go

```diff
@@ -565,8 +553,8 @@ func (n Node) Edit(i EditInput) {
 }
 
 // Content returns node's source code from input as a string
-func (n Node) Content(input []byte) string {
-	return string(input[n.StartByte():n.EndByte()])
+func (n Node) Content() string {
```

Comment:
If we keep the argument but allow it to be `nil`, in which case it defaults to `n.t.input`, then we can keep the API the same. Then, we could propose it upstream in a form that wouldn't require a major version bump.

Definitely not a blocker or anything, just a consideration.